### DISCUSSION TITLE
如何在IDEA中导入和运行Pekko项目

### DISCUSSION THREAD

**Discussion by He-Pin** [apache/pekko#884] [General]
欢迎欢迎

# 1. 克隆项目
![image](https://github.com/apache/incubator-pekko/assets/501740/f24a7445-234a-49f1-b25d-648c358c159d)

# 2. 设置好 SBT
设置好 sbt，不然你没法运行测试和正确的导入。

- 建议你现在使用 Java 8
- 建议按照图片中的设置

![image](https://github.com/apache/incubator-pekko/assets/501740/072e48aa-1165-4357-b8f1-64ab33e4a307)

# 3. 刷新工程
![image](https://github.com/apache/incubator-pekko/assets/501740/fba6495b-2043-4fdf-b3ca-ae513dfa3ba8)

# 4. 探索项目
现在你应该OK了，可以自主探索项目和跑单元测试了。比如：
编译时间可能比较长，耐心等待。

![image](https://github.com/apache/incubator-pekko/assets/501740/830458cd-90bf-486e-9efa-6775d14b301e)

觉得不错请点赞吧。

**Comment by jxnu-liguobin** [comment]
牛B！
